src/Microsoft.ComponentDetection.Detectors/cocoapods/PodComponentDetector.cs

```diff
@@ -405,5 +407,17 @@ private static (Pod pod, string key, DetectedComponent detectedComponent)[] Read
             })
             .ToArray();
         }
+
+        private static string NormalizePodfileGitUri(string gitOption)
+        {
+            // Podfiles can be built using git@ references to .git files, but this is not a valid Uri
+            // schema. Normalize to https:// so Uri creation doesn't fail
+            if (gitOption.StartsWith("git@", StringComparison.OrdinalIgnoreCase))
+            {
+                return $"https://{gitOption[4..]}";
```

Comment:
nit: Why not using [substring](https://docs.microsoft.com/en-us/dotnet/api/system.string.substring?view=net-6.0)? IMO it makes it more readable.